src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const tBody = document.querySelector('tbody');
+
+const fnAppendRow = function() {
+  if (tBody.children.length <= 10) {
```

Comment:
it's possible to add 11 rows/columns. please fix thee check here
![image](https://user-images.githubusercontent.com/22824431/222153757-20a3b993-8533-4126-afd0-9daa11624678.png)

```suggestion
  if (tBody.children.length < 10) {
```